### ISSUE TITLE
HTML Report: Visual cues for color-blind people

### ISSUE THREAD

**Issue by jernst** [fediverse-devnet/feditest#416]
From Accessibility Report:

> Besides color for links, also use another method such as text or an icon so that the colors can be distinguished for people who are color blind. Now the color difference between hovering on a link is very minimal. It should be a greater color difference.

Reference: https://www.w3.org/WAI/WCAG22/Understanding/use-of-color
